smtp_users.tf

```diff
@@ -1,5 +1,6 @@
 // Create IAM users to send mail
 module "smtp_users" {
+  #checkov:skip=CKV_AWS_273: An IAM user is required in this setup, cannot be moved to AWS SSO since this is not a human user
```

Comment:
Perhaps want to mention it's not possible to use an IAM role here? IAM user is the only option for SMTP auth.

Reply:
done